.github/workflows/build-ale-py.yml

```diff
@@ -0,0 +1,83 @@
+name: Build ale-py
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    name: Build ale-py on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      name: Install Python ${{ matrix.python_version }}
+      with:
+        python-version: "3.7"
+    - name: Install dependencies
+      run: |
+        python -m pip install git+https://github.com/mayeut/cibuildwheel.git@github-workflows
```

Comment:
We need to wait until there is an official release of `cibuildwheel`. This is currently their Github actions branch. It seems like it should be happening in the next couple weeks.

Reply:
Hi @JesseFarebro , can this be merged?

Reply:
Not yet, still waiting for a release of `cibuildwheel`. I'll make sure to let you know when this can be merged.